src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure implements AreaCalculetion {
+
+    private double baseB;
+    private double baseA;
+    private double height;
+
+    public IsoscelesTrapezoid(
+            double height,
+            double baseA,
+            double baseB,
+            String color,
+            String name
+    ) {
+        this.height = height;
+        this.baseA = baseA;
+        this.baseB = baseB;
+    }

```

Comment:
You are missing the call to the superclass constructor to initialize the `color` field. It should be `super(color);`.